docs/app/content-collections.md

```diff
@@ -1,168 +1,123 @@
 # Collection Page
 
-> This is the main gateway to your content items, providing a highly configurable Layout for browsing and visualizing
-> the items within a collection. The header of this page includes key action buttons for creating, deleting, and batch
-> editing items.
+> The Collection Page displays all Items within a Collection and comes with highly configurable Layouts for browsing,
+> visualizing, and managing these Items.
 
 [[toc]]
 
-## Creating an Item
+## Adjust Page Layout
 
-1. Navigate to the collection where you want to add an item
-2. Click on the "Create Item" (+ icon) button located in the header
-3. Complete the Item Detail form
-4. Click the "Save Item" (✓ icon) button located in the header
+<video autoplay muted loop controls title="Layouts">
+	<source src="https://cdn.directus.io/docs/v9/app-guide/content/content-collections/content-collections-20220415A/adjust-page-layout-20220415A.mp4" type="video/mp4" />
+</video>
 
-::: tip Singletons
+Layouts determine how you view or interact with Items in a Collection. Whether your Collection contains blog posts,
+project management tasks, geo-coordinates, temporal data or beyond; Layouts display Items in a logical and intuitive
```

Comment:
I was thinking with the videos that show click-of-a-button Layout adjustments it would seem intuitive enough.
But, changed it to "Layouts present Items in a more human-friendly way".
Is that toned-down enough?

Reply:
Yup! As long as we make sure we don't make it sound like "Hey this is easy, and therefore you're stupid for needing to read this doc"